src/screens/SendSheet.js

```diff
@@ -458,7 +470,11 @@ export default function SendSheet(props) {
       };
 
       try {
-        const signableTransaction = await createSignableTransaction(txDetails);
+        const signableTransaction = await performanceTracking.getState().executeFn({
+          fn: createSignableTransaction,
+          operation: TimeToSignOperation.SignTransaction,
```

Comment:
just a heads up that the `createSignableTransaction` doesn't sign the transaction itself, it basically just creates the txn payload data - so this will be a bit discrepant from the other places we use `TimeToSignOperation.SignTransaction` which actually load the wallet and sign the txn. The `sendTransaction` below is where the signing and broadcasting will happen, so we may be covered just from that.